_posts/2018-10-11-interface-power.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title:  Interface Power
+date:   2018-10-11
+category: technology
+author: frost-fox
+---
+
+### Abstraction and polymorphism
+
+In object oriented programming such principles as Polymorphism and Abstraction are considered as golden rules, and we
+all understand theoretical basis of this principles and their simplistic beauty. Abstraction allows us to create 
+functions defined as **abstract**, without specifying it's details at the beginning and deferring this moment to 
```

Comment:
Chyba to zbyt szybki przeskok od „abstrakcji” do „abstrakcyjnej klasy / metody”…